src/commands/set-shas.yml

```diff
@@ -41,6 +41,7 @@ steps:
         PARAM_ALLOW_ON_HOLD: <<parameters.allow-on-hold-workflow>>
         PARAM_SKIP_BRANCH_FILTER: <<parameters.skip-branch-filter>>
         PARAM_SCRIPT: <<include(scripts/find-successful-workflow.js)>>
+        PARAM_SCRIPT_PATH: "index.js"
```

Comment:
There's not much point in adding this parameter since it will never be changed.

Reply:
👍 Removed the parameter commit